 WARN [main] 2017-04-09 21:24:06,949 URLConfigurationSource.java (line 122) No URLs will be polled as dynamic configuration sources.
 INFO [main] 2017-04-09 21:24:07,108 URLConfigurationSource.java (line 123) To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
 INFO [main] 2017-04-09 21:24:07,127 DynamicPropertyFactory.java (line 271) DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@3cbbc1e0
 INFO [main] 2017-04-09 21:24:07,239 KaryonServer.java (line 146) Loading application properties with app id: edge and environment: null
 INFO [main] 2017-04-09 21:24:07,289 ConfigurationUtils.java (line 182) Loaded properties file jar:file:/home/polivier/Documents/AL/recipes-rss/rss-edge/build/libs/rss-edge-0.1.0-SNAPSHOT.jar!/edge.properties
 INFO [main] 2017-04-09 21:24:09,510 Version.java (line 56) Hibernate Validator null
 INFO [main] 2017-04-09 21:24:09,532 ServerBootstrap.java (line 239) No health check invocation strategy specified, using the default strategy com.netflix.karyon.server.eureka.AsyncHealthCheckInvocationStrategy. In order to override this behavior you provide an implementation of com.netflix.karyon.server.eureka.HealthCheckInvocationStrategy and specify the fully qualified class name of the implementation in the property com.netflix.karyon.health.check.strategy
 INFO [main] 2017-04-09 21:24:09,592 AsyncHealthCheckInvocationStrategy.java (line 77) Application health check handler to be used by karyon: com.netflix.recipes.rss.server.EdgeHealthCheckHandler
 INFO [main] 2017-04-09 21:24:09,627 NativeMethodAccessorImpl.java (line -2) Logging to org.slf4j.impl.Log4jLoggerAdapter(org.mortbay.log) via org.mortbay.log.Slf4jLog
 WARN [main] 2017-04-09 21:24:09,729 PropertiesInstanceConfig.java (line 338) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 WARN [main] 2017-04-09 21:24:09,730 DefaultEurekaClientConfig.java (line 89) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 INFO [main] 2017-04-09 21:24:10,328 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [main] 2017-04-09 21:24:10,329 DiscoveryClient.java (line 555) Application is null : false
 INFO [main] 2017-04-09 21:24:10,330 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [main] 2017-04-09 21:24:10,331 DiscoveryClient.java (line 560) Application version is -1: true
ERROR [main] 2017-04-09 21:24:10,397 DiscoveryClient.java (line 847) Can't get a response from http://localhost/eureka/v2/apps/
Can't contact any eureka nodes - possibly a security group issue?
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.<init>(DiscoveryClient.java:199)
	at com.netflix.discovery.DiscoveryManager.initComponent(DiscoveryManager.java:73)
	at com.netflix.karyon.server.eureka.EurekaHandler.register(EurekaHandler.java:115)
	at com.netflix.karyon.server.lifecycle.ServerInitializer.initialize(ServerInitializer.java:69)
	at com.netflix.karyon.server.KaryonServer.start(KaryonServer.java:189)
	at com.netflix.recipes.rss.server.BaseJettyServer.start(BaseJettyServer.java:75)
	at com.netflix.recipes.rss.server.EdgeServer.main(EdgeServer.java:50)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 17 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 24 more
ERROR [main] 2017-04-09 21:24:10,399 DiscoveryClient.java (line 590) DiscoveryClient_UNKNOWN/pc54.home - was unable to refresh it's cache! status = org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.<init>(DiscoveryClient.java:199)
	at com.netflix.discovery.DiscoveryManager.initComponent(DiscoveryManager.java:73)
	at com.netflix.karyon.server.eureka.EurekaHandler.register(EurekaHandler.java:115)
	at com.netflix.karyon.server.lifecycle.ServerInitializer.initialize(ServerInitializer.java:69)
	at com.netflix.karyon.server.KaryonServer.start(KaryonServer.java:189)
	at com.netflix.recipes.rss.server.BaseJettyServer.start(BaseJettyServer.java:75)
	at com.netflix.recipes.rss.server.EdgeServer.main(EdgeServer.java:50)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 17 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 24 more
 INFO [main] 2017-04-09 21:24:10,401 DiscoveryClient.java (line 886) Starting heartbeat executor: renew interval is: 30
 INFO [main] 2017-04-09 21:24:10,418 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-09 21:24:10,494 Server.java (line 266) jetty-7.x.y-SNAPSHOT
 INFO [main] 2017-04-09 21:24:10,540 ContextHandler.java (line 744) started o.e.j.s.ServletContextHandler{/,null}
 INFO [main] 2017-04-09 21:24:10,579 AbstractConnector.java (line 338) Started SelectChannelConnector@0.0.0.0:9092
 INFO [main] 2017-04-09 21:24:10,581 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-09 21:24:10,582 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-09 21:24:10,589 WebAdminComponent.java (line 46) Set the default page for admin container to: /adminres/webadmin/index.html
 INFO [main] 2017-04-09 21:24:10,590 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-09 21:24:10,595 ApplicationFinder.java (line 88) No application classes (Annotated with @Application) found. It is fine if you do not use the annotation model for applications.
 INFO [main] 2017-04-09 21:24:10,596 ServerInitializer.java (line 86) No application classes found, this is fine if the application class is not annotated.
 INFO [main] 2017-04-09 21:24:10,597 NativeMethodAccessorImpl.java (line -2) jetty-6.0.x
 INFO [main] 2017-04-09 21:24:10,707 NativeMethodAccessorImpl.java (line -2) Started SocketConnector @ 0.0.0.0:9090
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:24:40,401 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:24:40,402 DiscoveryClient.java (line 555) Application is null : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:24:40,402 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:24:40,402 DiscoveryClient.java (line 560) Application version is -1: true
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:24:40,405 DiscoveryClient.java (line 847) Can't get a response from http://localhost/eureka/v2/apps/
Can't contact any eureka nodes - possibly a security group issue?
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:24:40,406 DiscoveryClient.java (line 590) DiscoveryClient_UNKNOWN/pc54.home - was unable to refresh it's cache! status = org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:24:50,403 DiscoveryClient.java (line 1283) DiscoveryClient_UNKNOWN/pc54.home - retransmit instance info with status UP
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:24:50,403 DiscoveryClient.java (line 424) DiscoveryClient_UNKNOWN/pc54.home: registering service...
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:24:50,404 DiscoveryClient.java (line 429) DiscoveryClient_UNKNOWN/pc54.home - registration status: not sent
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:10,401 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:10,402 DiscoveryClient.java (line 555) Application is null : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:10,402 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:10,402 DiscoveryClient.java (line 560) Application version is -1: true
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:10,404 DiscoveryClient.java (line 847) Can't get a response from http://localhost/eureka/v2/apps/
Can't contact any eureka nodes - possibly a security group issue?
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:10,404 DiscoveryClient.java (line 590) DiscoveryClient_UNKNOWN/pc54.home - was unable to refresh it's cache! status = org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:40,401 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:40,401 DiscoveryClient.java (line 555) Application is null : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:40,401 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:40,402 DiscoveryClient.java (line 560) Application version is -1: true
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:40,403 DiscoveryClient.java (line 847) Can't get a response from http://localhost/eureka/v2/apps/
Can't contact any eureka nodes - possibly a security group issue?
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:25:40,403 DiscoveryClient.java (line 590) DiscoveryClient_UNKNOWN/pc54.home - was unable to refresh it's cache! status = org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:10,402 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:10,402 DiscoveryClient.java (line 555) Application is null : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:10,402 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:10,402 DiscoveryClient.java (line 560) Application version is -1: true
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:10,404 DiscoveryClient.java (line 847) Can't get a response from http://localhost/eureka/v2/apps/
Can't contact any eureka nodes - possibly a security group issue?
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:10,405 DiscoveryClient.java (line 590) DiscoveryClient_UNKNOWN/pc54.home - was unable to refresh it's cache! status = org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:40,401 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:40,402 DiscoveryClient.java (line 555) Application is null : false
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:40,402 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:40,402 DiscoveryClient.java (line 560) Application version is -1: true
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:40,404 DiscoveryClient.java (line 847) Can't get a response from http://localhost/eureka/v2/apps/
Can't contact any eureka nodes - possibly a security group issue?
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
ERROR [DiscoveryClient_CacheRefresher] 2017-04-09 21:26:40,404 DiscoveryClient.java (line 590) DiscoveryClient_UNKNOWN/pc54.home - was unable to refresh it's cache! status = org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:120)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.discovery.DiscoveryClient.getUrl(DiscoveryClient.java:1196)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:800)
	at com.netflix.discovery.DiscoveryClient.makeRemoteCall(DiscoveryClient.java:753)
	at com.netflix.discovery.DiscoveryClient.getAndStoreFullRegistry(DiscoveryClient.java:615)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:562)
	at com.netflix.discovery.DiscoveryClient.access$1200(DiscoveryClient.java:87)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1317)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://localhost refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 14 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 21 more
 WARN [main] 2017-04-09 21:29:10,040 URLConfigurationSource.java (line 122) No URLs will be polled as dynamic configuration sources.
 INFO [main] 2017-04-09 21:29:10,238 URLConfigurationSource.java (line 123) To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
 INFO [main] 2017-04-09 21:29:10,277 DynamicPropertyFactory.java (line 271) DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@3cbbc1e0
 INFO [main] 2017-04-09 21:29:10,406 KaryonServer.java (line 146) Loading application properties with app id: edge and environment: null
 INFO [main] 2017-04-09 21:29:10,451 ConfigurationUtils.java (line 182) Loaded properties file jar:file:/home/polivier/Documents/AL/recipes-rss/rss-edge/build/libs/rss-edge-0.1.0-SNAPSHOT.jar!/edge.properties
 INFO [main] 2017-04-09 21:29:12,678 Version.java (line 56) Hibernate Validator null
 INFO [main] 2017-04-09 21:29:12,699 ServerBootstrap.java (line 239) No health check invocation strategy specified, using the default strategy com.netflix.karyon.server.eureka.AsyncHealthCheckInvocationStrategy. In order to override this behavior you provide an implementation of com.netflix.karyon.server.eureka.HealthCheckInvocationStrategy and specify the fully qualified class name of the implementation in the property com.netflix.karyon.health.check.strategy
 INFO [main] 2017-04-09 21:29:12,747 AsyncHealthCheckInvocationStrategy.java (line 77) Application health check handler to be used by karyon: com.netflix.recipes.rss.server.EdgeHealthCheckHandler
 INFO [main] 2017-04-09 21:29:12,773 NativeMethodAccessorImpl.java (line -2) Logging to org.slf4j.impl.Log4jLoggerAdapter(org.mortbay.log) via org.mortbay.log.Slf4jLog
 WARN [main] 2017-04-09 21:29:12,892 PropertiesInstanceConfig.java (line 338) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 WARN [main] 2017-04-09 21:29:12,894 DefaultEurekaClientConfig.java (line 89) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 INFO [main] 2017-04-09 21:29:13,599 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [main] 2017-04-09 21:29:13,600 DiscoveryClient.java (line 555) Application is null : false
 INFO [main] 2017-04-09 21:29:13,600 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [main] 2017-04-09 21:29:13,601 DiscoveryClient.java (line 560) Application version is -1: true
 INFO [main] 2017-04-09 21:29:13,711 DiscoveryClient.java (line 616) Getting all instance registry info from the eureka server
 INFO [main] 2017-04-09 21:29:13,909 DiscoveryClient.java (line 624) The response status is 200
 INFO [main] 2017-04-09 21:29:13,911 DiscoveryClient.java (line 886) Starting heartbeat executor: renew interval is: 30
 INFO [main] 2017-04-09 21:29:13,929 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-09 21:29:14,006 Server.java (line 266) jetty-7.x.y-SNAPSHOT
 INFO [main] 2017-04-09 21:29:14,059 ContextHandler.java (line 744) started o.e.j.s.ServletContextHandler{/,null}
 INFO [main] 2017-04-09 21:29:14,096 AbstractConnector.java (line 338) Started SelectChannelConnector@0.0.0.0:9092
 INFO [main] 2017-04-09 21:29:14,099 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-09 21:29:14,100 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-09 21:29:14,103 WebAdminComponent.java (line 46) Set the default page for admin container to: /adminres/webadmin/index.html
 INFO [main] 2017-04-09 21:29:14,104 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-09 21:29:14,104 ApplicationFinder.java (line 88) No application classes (Annotated with @Application) found. It is fine if you do not use the annotation model for applications.
 INFO [main] 2017-04-09 21:29:14,105 ServerInitializer.java (line 86) No application classes found, this is fine if the application class is not annotated.
 INFO [main] 2017-04-09 21:29:14,105 NativeMethodAccessorImpl.java (line -2) jetty-6.0.x
 INFO [main] 2017-04-09 21:29:14,302 NativeMethodAccessorImpl.java (line -2) Started SocketConnector @ 0.0.0.0:9090
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:29:53,912 DiscoveryClient.java (line 1283) DiscoveryClient_UNKNOWN/pc54.home - retransmit instance info with status UP
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:29:53,913 DiscoveryClient.java (line 424) DiscoveryClient_UNKNOWN/pc54.home: registering service...
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:29:53,913 DiscoveryClient.java (line 429) DiscoveryClient_UNKNOWN/pc54.home - registration status: not sent
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:30:30,370 ConnectionPoolCleaner.java (line 110) Initializing ConnectionPoolCleaner for NFHttpClient:middletier-client
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:30:30,592 BaseLoadBalancer.java (line 184) Client:middletier-client instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=middletier-client,current list of Servers=[],Load balancer stats=Zone stats: {},Server stats: []}ServerList:null
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:30:30,623 ClientFactory.java (line 161) Client:middletier-client instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=middletier-client,current list of Servers=[pc54.home:9191],Load balancer stats=Zone stats: {defaultzone=[Zone:defaultzone;	Instance count:1;	Active connections count: 0;	Circuit breaker tripped count: 0;	Active connections per server: 0.0;]
},Server stats: [[Server:pc54.home:9191;	Zone:defaultZone;	Total Requests:0;	Successive connection failure:0;	Total blackout seconds:0;	Last connection made:Thu Jan 01 01:00:00 CET 1970;	First connection made: Thu Jan 01 01:00:00 CET 1970;	Active Connections:0;	total failure count in last (1000) msecs:0;	average resp time:0.0;	90 percentile resp time:0.0;	95 percentile resp time:0.0;	min resp time:0.0;	max resp time:0.0;	stddev resp time:0.0]
]}ServerList:DiscoveryEnabledNIWSServerList:; clientName:middletier-client; Effective vipAddresses:middletier; isSecure:false; datacenter:null
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:30:30,629 ClientFactory.java (line 86) Client Registered:com.netflix.niws.client.http.RestClient@105f9569
ERROR [hystrix-RSSThreadPool-1] 2017-04-09 21:30:30,644 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-1] 2017-04-09 21:30:30,847 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-1] 2017-04-09 21:30:30,849 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-1] 2017-04-09 21:30:31,051 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-2] 2017-04-09 21:32:01,236 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default?url=http://rss.cnn.com/rss/edition.rss
ERROR [hystrix-RSSThreadPool-2] 2017-04-09 21:32:01,438 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception occurred when adding a RSS feed
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:73)
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default?url=http://rss.cnn.com/rss/edition.rss
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:69)
	... 9 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.post(WebResource.java:568)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:501)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-3] 2017-04-09 21:32:01,442 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-3] 2017-04-09 21:32:01,643 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-3] 2017-04-09 21:32:01,644 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-3] 2017-04-09 21:32:01,846 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-4] 2017-04-09 21:32:01,856 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-4] 2017-04-09 21:32:02,058 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-4] 2017-04-09 21:32:02,060 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-4] 2017-04-09 21:32:02,261 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-5] 2017-04-09 21:32:15,454 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default?url=http://rss.cnn.com/rss/edition.rss
ERROR [hystrix-RSSThreadPool-5] 2017-04-09 21:32:15,656 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception occurred when adding a RSS feed
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:73)
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default?url=http://rss.cnn.com/rss/edition.rss
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:69)
	... 9 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.post(WebResource.java:568)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:501)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-6] 2017-04-09 21:32:15,664 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-6] 2017-04-09 21:32:15,866 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-6] 2017-04-09 21:32:15,867 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-6] 2017-04-09 21:32:16,069 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-7] 2017-04-09 21:32:16,071 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-7] 2017-04-09 21:32:16,273 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-7] 2017-04-09 21:32:16,274 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-7] 2017-04-09 21:32:16,475 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-8] 2017-04-09 21:32:50,329 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default?url=http://news.yahoo.com/rss/us
ERROR [hystrix-RSSThreadPool-8] 2017-04-09 21:32:50,530 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception occurred when adding a RSS feed
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:73)
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default?url=http://news.yahoo.com/rss/us
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:69)
	... 9 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.post(WebResource.java:568)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:501)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-9] 2017-04-09 21:32:50,533 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-9] 2017-04-09 21:32:50,734 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-9] 2017-04-09 21:32:50,736 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-9] 2017-04-09 21:32:50,937 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:32:50,940 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:32:51,141 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:32:51,143 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:32:51,344 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:10,247 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default?url=http://news.yahoo.com/rss/us
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:10,448 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception occurred when adding a RSS feed
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:73)
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default?url=http://news.yahoo.com/rss/us
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	at com.netflix.recipes.rss.hystrix.AddRSSCommand.run(AddRSSCommand.java:69)
	... 9 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.post(WebResource.java:568)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:501)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:10,453 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:10,655 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:10,659 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:10,861 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:10,863 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:11,064 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:11,066 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:11,267 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:21,490 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:21,691 AbstractLoadBalancerAwareClient.java (line 420) Exception while executing request which is deemed retry-able, retrying ..., Next Server Retry Attempt#:1, URI tried:/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:21,692 AbstractLoadBalancerAwareClient.java (line 298) Exception while executing request which is deemed retry-able, retrying ..., SAME Server Retry Attempt#:1, URI:http://pc54.home:9191/middletier/rss/user/default
ERROR [hystrix-RSSThreadPool-10] 2017-04-09 21:33:21,894 HystrixCommand.java (line 812) Error executing HystrixCommand
java.lang.RuntimeException: Exception
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:66)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:36)
	at com.netflix.hystrix.HystrixCommand.executeCommand(HystrixCommand.java:764)
	at com.netflix.hystrix.HystrixCommand.access$1400(HystrixCommand.java:81)
	at com.netflix.hystrix.HystrixCommand$2.call(HystrixCommand.java:706)
	at com.netflix.hystrix.strategy.concurrency.HystrixContextCallable.call(HystrixContextCallable.java:45)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.netflix.client.ClientException: NUMBER_OF_RETRIES_NEXTSERVER_EXCEEDED :1 retries, while making a RestClient call for:/middletier/rss/user/default:Connexion refusée (Connection refused)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:413)
	at com.netflix.recipes.rss.hystrix.GetRSSCommand.run(GetRSSCommand.java:62)
	... 9 more
Caused by: com.netflix.client.ClientException: NUMBEROFRETRIESEXEEDED :1 retries, while making a RestClient call for:http://pc54.home:9191/middletier/rss/user/default
	at com.netflix.client.AbstractLoadBalancerAwareClient.handleRetry(AbstractLoadBalancerAwareClient.java:294)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:201)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeWithLoadBalancer(AbstractLoadBalancerAwareClient.java:402)
	... 10 more
Caused by: com.sun.jersey.api.client.ClientHandlerException: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:184)
	at com.sun.jersey.api.client.Client.handle(Client.java:648)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:680)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:507)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:498)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:418)
	at com.netflix.niws.client.http.RestClient.execute(RestClient.java:78)
	at com.netflix.client.AbstractLoadBalancerAwareClient.executeOnSingleServer(AbstractLoadBalancerAwareClient.java:192)
	... 11 more
Caused by: org.apache.http.conn.HttpHostConnectException: Connection to http://pc54.home:9191 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:776)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:170)
	... 19 more
Caused by: java.net.ConnectException: Connexion refusée (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 26 more
 WARN [main] 2017-04-09 21:34:58,516 URLConfigurationSource.java (line 122) No URLs will be polled as dynamic configuration sources.
 INFO [main] 2017-04-09 21:34:58,711 URLConfigurationSource.java (line 123) To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
 INFO [main] 2017-04-09 21:34:58,736 DynamicPropertyFactory.java (line 271) DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@3cbbc1e0
 INFO [main] 2017-04-09 21:34:58,901 KaryonServer.java (line 146) Loading application properties with app id: edge and environment: null
 INFO [main] 2017-04-09 21:34:58,949 ConfigurationUtils.java (line 182) Loaded properties file jar:file:/home/polivier/Documents/AL/recipes-rss/rss-edge/build/libs/rss-edge-0.1.0-SNAPSHOT.jar!/edge.properties
 INFO [main] 2017-04-09 21:35:01,497 Version.java (line 56) Hibernate Validator null
 INFO [main] 2017-04-09 21:35:01,514 ServerBootstrap.java (line 239) No health check invocation strategy specified, using the default strategy com.netflix.karyon.server.eureka.AsyncHealthCheckInvocationStrategy. In order to override this behavior you provide an implementation of com.netflix.karyon.server.eureka.HealthCheckInvocationStrategy and specify the fully qualified class name of the implementation in the property com.netflix.karyon.health.check.strategy
 INFO [main] 2017-04-09 21:35:01,541 AsyncHealthCheckInvocationStrategy.java (line 77) Application health check handler to be used by karyon: com.netflix.recipes.rss.server.EdgeHealthCheckHandler
 INFO [main] 2017-04-09 21:35:01,580 NativeMethodAccessorImpl.java (line -2) Logging to org.slf4j.impl.Log4jLoggerAdapter(org.mortbay.log) via org.mortbay.log.Slf4jLog
 WARN [main] 2017-04-09 21:35:01,679 PropertiesInstanceConfig.java (line 338) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 WARN [main] 2017-04-09 21:35:01,682 DefaultEurekaClientConfig.java (line 89) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 INFO [main] 2017-04-09 21:35:02,484 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [main] 2017-04-09 21:35:02,484 DiscoveryClient.java (line 555) Application is null : false
 INFO [main] 2017-04-09 21:35:02,485 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [main] 2017-04-09 21:35:02,485 DiscoveryClient.java (line 560) Application version is -1: true
 INFO [main] 2017-04-09 21:35:02,673 DiscoveryClient.java (line 616) Getting all instance registry info from the eureka server
 INFO [main] 2017-04-09 21:35:03,047 DiscoveryClient.java (line 624) The response status is 200
 INFO [main] 2017-04-09 21:35:03,048 DiscoveryClient.java (line 886) Starting heartbeat executor: renew interval is: 30
 INFO [main] 2017-04-09 21:35:03,091 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-09 21:35:03,197 Server.java (line 266) jetty-7.x.y-SNAPSHOT
 INFO [main] 2017-04-09 21:35:03,236 ContextHandler.java (line 744) started o.e.j.s.ServletContextHandler{/,null}
 INFO [main] 2017-04-09 21:35:03,312 AbstractConnector.java (line 338) Started SelectChannelConnector@0.0.0.0:9092
 INFO [main] 2017-04-09 21:35:03,313 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-09 21:35:03,314 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-09 21:35:03,317 WebAdminComponent.java (line 46) Set the default page for admin container to: /adminres/webadmin/index.html
 INFO [main] 2017-04-09 21:35:03,318 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-09 21:35:03,318 ApplicationFinder.java (line 88) No application classes (Annotated with @Application) found. It is fine if you do not use the annotation model for applications.
 INFO [main] 2017-04-09 21:35:03,319 ServerInitializer.java (line 86) No application classes found, this is fine if the application class is not annotated.
 INFO [main] 2017-04-09 21:35:03,319 NativeMethodAccessorImpl.java (line -2) jetty-6.0.x
 INFO [main] 2017-04-09 21:35:03,546 NativeMethodAccessorImpl.java (line -2) Started SocketConnector @ 0.0.0.0:9090
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:35:32,151 ConnectionPoolCleaner.java (line 110) Initializing ConnectionPoolCleaner for NFHttpClient:middletier-client
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:35:32,383 BaseLoadBalancer.java (line 184) Client:middletier-client instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=middletier-client,current list of Servers=[],Load balancer stats=Zone stats: {},Server stats: []}ServerList:null
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:35:32,419 ClientFactory.java (line 161) Client:middletier-client instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=middletier-client,current list of Servers=[pc54.home:9191],Load balancer stats=Zone stats: {defaultzone=[Zone:defaultzone;	Instance count:1;	Active connections count: 0;	Circuit breaker tripped count: 0;	Active connections per server: 0.0;]
},Server stats: [[Server:pc54.home:9191;	Zone:defaultZone;	Total Requests:0;	Successive connection failure:0;	Total blackout seconds:0;	Last connection made:Thu Jan 01 01:00:00 CET 1970;	First connection made: Thu Jan 01 01:00:00 CET 1970;	Active Connections:0;	total failure count in last (1000) msecs:0;	average resp time:0.0;	90 percentile resp time:0.0;	95 percentile resp time:0.0;	min resp time:0.0;	max resp time:0.0;	stddev resp time:0.0]
]}ServerList:DiscoveryEnabledNIWSServerList:; clientName:middletier-client; Effective vipAddresses:middletier; isSecure:false; datacenter:null
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:35:32,426 ClientFactory.java (line 86) Client Registered:com.netflix.niws.client.http.RestClient@ffa7f84
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:35:43,051 DiscoveryClient.java (line 1283) DiscoveryClient_UNKNOWN/pc54.home - retransmit instance info with status UP
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:35:43,051 DiscoveryClient.java (line 424) DiscoveryClient_UNKNOWN/pc54.home: registering service...
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:35:43,051 DiscoveryClient.java (line 429) DiscoveryClient_UNKNOWN/pc54.home - registration status: not sent
 WARN [btpool0-2] 2017-04-09 21:36:37,057 NativeMethodAccessorImpl.java (line -2) EXCEPTION 
javax.servlet.ServletException: org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.apache.jasper.runtime.PageContextImpl.doHandlePageException(PageContextImpl.java:922)
	at org.apache.jasper.runtime.PageContextImpl.handlePageException(PageContextImpl.java:855)
	at org.apache.jsp.jsp.rss_jsp._jspService(Unknown Source)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:373)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:464)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:358)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:459)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:360)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:231)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:629)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:141)
	at org.mortbay.jetty.Server.handle(Server.java:303)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:452)
	at org.mortbay.jetty.HttpConnection$RequestHandler.content(HttpConnection.java:735)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:636)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:209)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:349)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:217)
	at org.mortbay.thread.BoundedThreadPool$PoolThread.run(BoundedThreadPool.java:475)
Caused by: org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONObject.<init>(JSONObject.java:179)
	at org.json.JSONObject.<init>(JSONObject.java:402)
	... 20 more
 WARN [btpool0-2] 2017-04-09 21:36:37,058 NativeMethodAccessorImpl.java (line -2) Nested in javax.servlet.ServletException: org.json.JSONException: A JSONObject text must begin with '{' at character 0:
org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONObject.<init>(JSONObject.java:179)
	at org.json.JSONObject.<init>(JSONObject.java:402)
	at org.apache.jsp.jsp.rss_jsp._jspService(Unknown Source)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:373)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:464)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:358)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:459)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:360)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:231)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:629)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:141)
	at org.mortbay.jetty.Server.handle(Server.java:303)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:452)
	at org.mortbay.jetty.HttpConnection$RequestHandler.content(HttpConnection.java:735)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:636)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:209)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:349)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:217)
	at org.mortbay.thread.BoundedThreadPool$PoolThread.run(BoundedThreadPool.java:475)
 WARN [btpool0-2] 2017-04-09 21:36:37,059 NativeMethodAccessorImpl.java (line -2) /jsp/rss.jsp: 
org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONObject.<init>(JSONObject.java:179)
	at org.json.JSONObject.<init>(JSONObject.java:402)
	at org.apache.jsp.jsp.rss_jsp._jspService(Unknown Source)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:373)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:464)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:358)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:459)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:360)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:231)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:629)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:141)
	at org.mortbay.jetty.Server.handle(Server.java:303)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:452)
	at org.mortbay.jetty.HttpConnection$RequestHandler.content(HttpConnection.java:735)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:636)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:209)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:349)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:217)
	at org.mortbay.thread.BoundedThreadPool$PoolThread.run(BoundedThreadPool.java:475)
 WARN [btpool0-2] 2017-04-09 21:36:37,059 NativeMethodAccessorImpl.java (line -2) /jsp/rss.jsp
org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONObject.<init>(JSONObject.java:179)
	at org.json.JSONObject.<init>(JSONObject.java:402)
	at org.apache.jsp.jsp.rss_jsp._jspService(Unknown Source)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:373)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:464)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:358)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:459)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:360)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:231)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:629)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:141)
	at org.mortbay.jetty.Server.handle(Server.java:303)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:452)
	at org.mortbay.jetty.HttpConnection$RequestHandler.content(HttpConnection.java:735)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:636)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:209)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:349)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:217)
	at org.mortbay.thread.BoundedThreadPool$PoolThread.run(BoundedThreadPool.java:475)
 WARN [btpool0-2] 2017-04-09 21:36:37,868 NativeMethodAccessorImpl.java (line -2) EXCEPTION 
javax.servlet.ServletException: org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.apache.jasper.runtime.PageContextImpl.doHandlePageException(PageContextImpl.java:922)
	at org.apache.jasper.runtime.PageContextImpl.handlePageException(PageContextImpl.java:855)
	at org.apache.jsp.jsp.rss_jsp._jspService(Unknown Source)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:373)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:464)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:358)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:459)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:360)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:231)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:629)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:141)
	at org.mortbay.jetty.Server.handle(Server.java:303)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:452)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:721)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:509)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:209)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:349)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:217)
	at org.mortbay.thread.BoundedThreadPool$PoolThread.run(BoundedThreadPool.java:475)
Caused by: org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONObject.<init>(JSONObject.java:179)
	at org.json.JSONObject.<init>(JSONObject.java:402)
	... 20 more
 WARN [btpool0-2] 2017-04-09 21:36:37,868 NativeMethodAccessorImpl.java (line -2) Nested in javax.servlet.ServletException: org.json.JSONException: A JSONObject text must begin with '{' at character 0:
org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONObject.<init>(JSONObject.java:179)
	at org.json.JSONObject.<init>(JSONObject.java:402)
	at org.apache.jsp.jsp.rss_jsp._jspService(Unknown Source)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:373)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:464)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:358)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:459)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:360)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:231)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:629)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:141)
	at org.mortbay.jetty.Server.handle(Server.java:303)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:452)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:721)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:509)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:209)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:349)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:217)
	at org.mortbay.thread.BoundedThreadPool$PoolThread.run(BoundedThreadPool.java:475)
 WARN [btpool0-2] 2017-04-09 21:36:37,869 NativeMethodAccessorImpl.java (line -2) /jsp/rss.jsp: 
org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONObject.<init>(JSONObject.java:179)
	at org.json.JSONObject.<init>(JSONObject.java:402)
	at org.apache.jsp.jsp.rss_jsp._jspService(Unknown Source)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:373)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:464)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:358)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:459)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:360)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:231)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:629)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:141)
	at org.mortbay.jetty.Server.handle(Server.java:303)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:452)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:721)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:509)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:209)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:349)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:217)
	at org.mortbay.thread.BoundedThreadPool$PoolThread.run(BoundedThreadPool.java:475)
 WARN [btpool0-2] 2017-04-09 21:36:37,869 NativeMethodAccessorImpl.java (line -2) /jsp/rss.jsp
org.json.JSONException: A JSONObject text must begin with '{' at character 0
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONObject.<init>(JSONObject.java:179)
	at org.json.JSONObject.<init>(JSONObject.java:402)
	at org.apache.jsp.jsp.rss_jsp._jspService(Unknown Source)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:373)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:464)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:358)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:459)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:360)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:231)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:629)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:141)
	at org.mortbay.jetty.Server.handle(Server.java:303)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:452)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:721)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:509)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:209)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:349)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:217)
	at org.mortbay.thread.BoundedThreadPool$PoolThread.run(BoundedThreadPool.java:475)
 WARN [main] 2017-04-09 21:37:21,780 URLConfigurationSource.java (line 122) No URLs will be polled as dynamic configuration sources.
 INFO [main] 2017-04-09 21:37:22,147 URLConfigurationSource.java (line 123) To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
 INFO [main] 2017-04-09 21:37:22,186 DynamicPropertyFactory.java (line 271) DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@3cbbc1e0
 INFO [main] 2017-04-09 21:37:22,404 KaryonServer.java (line 146) Loading application properties with app id: edge and environment: null
 INFO [main] 2017-04-09 21:37:22,483 ConfigurationUtils.java (line 182) Loaded properties file jar:file:/home/polivier/Documents/AL/recipes-rss/rss-edge/build/libs/rss-edge-0.1.0-SNAPSHOT.jar!/edge.properties
 INFO [main] 2017-04-09 21:37:24,707 Version.java (line 56) Hibernate Validator null
 INFO [main] 2017-04-09 21:37:24,727 ServerBootstrap.java (line 239) No health check invocation strategy specified, using the default strategy com.netflix.karyon.server.eureka.AsyncHealthCheckInvocationStrategy. In order to override this behavior you provide an implementation of com.netflix.karyon.server.eureka.HealthCheckInvocationStrategy and specify the fully qualified class name of the implementation in the property com.netflix.karyon.health.check.strategy
 INFO [main] 2017-04-09 21:37:24,760 AsyncHealthCheckInvocationStrategy.java (line 77) Application health check handler to be used by karyon: com.netflix.recipes.rss.server.EdgeHealthCheckHandler
 INFO [main] 2017-04-09 21:37:24,796 NativeMethodAccessorImpl.java (line -2) Logging to org.slf4j.impl.Log4jLoggerAdapter(org.mortbay.log) via org.mortbay.log.Slf4jLog
 WARN [main] 2017-04-09 21:37:24,871 PropertiesInstanceConfig.java (line 338) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 WARN [main] 2017-04-09 21:37:24,874 DefaultEurekaClientConfig.java (line 89) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 INFO [main] 2017-04-09 21:37:25,523 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [main] 2017-04-09 21:37:25,523 DiscoveryClient.java (line 555) Application is null : false
 INFO [main] 2017-04-09 21:37:25,523 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [main] 2017-04-09 21:37:25,525 DiscoveryClient.java (line 560) Application version is -1: true
 INFO [main] 2017-04-09 21:37:25,630 DiscoveryClient.java (line 616) Getting all instance registry info from the eureka server
 INFO [main] 2017-04-09 21:37:25,840 DiscoveryClient.java (line 624) The response status is 200
 INFO [main] 2017-04-09 21:37:25,842 DiscoveryClient.java (line 886) Starting heartbeat executor: renew interval is: 30
 INFO [main] 2017-04-09 21:37:25,862 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-09 21:37:25,938 Server.java (line 266) jetty-7.x.y-SNAPSHOT
 INFO [main] 2017-04-09 21:37:25,982 ContextHandler.java (line 744) started o.e.j.s.ServletContextHandler{/,null}
 INFO [main] 2017-04-09 21:37:26,018 AbstractConnector.java (line 338) Started SelectChannelConnector@0.0.0.0:9092
 INFO [main] 2017-04-09 21:37:26,022 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-09 21:37:26,023 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-09 21:37:26,027 WebAdminComponent.java (line 46) Set the default page for admin container to: /adminres/webadmin/index.html
 INFO [main] 2017-04-09 21:37:26,028 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-09 21:37:26,029 ApplicationFinder.java (line 88) No application classes (Annotated with @Application) found. It is fine if you do not use the annotation model for applications.
 INFO [main] 2017-04-09 21:37:26,029 ServerInitializer.java (line 86) No application classes found, this is fine if the application class is not annotated.
 INFO [main] 2017-04-09 21:37:26,030 NativeMethodAccessorImpl.java (line -2) jetty-6.0.x
 INFO [main] 2017-04-09 21:37:26,219 NativeMethodAccessorImpl.java (line -2) Started SocketConnector @ 0.0.0.0:9090
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:38:05,844 DiscoveryClient.java (line 1283) DiscoveryClient_UNKNOWN/pc54.home - retransmit instance info with status UP
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:38:05,844 DiscoveryClient.java (line 424) DiscoveryClient_UNKNOWN/pc54.home: registering service...
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-09 21:38:05,844 DiscoveryClient.java (line 429) DiscoveryClient_UNKNOWN/pc54.home - registration status: not sent
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:38:51,378 ConnectionPoolCleaner.java (line 110) Initializing ConnectionPoolCleaner for NFHttpClient:middletier-client
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:38:51,497 BaseLoadBalancer.java (line 184) Client:middletier-client instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=middletier-client,current list of Servers=[],Load balancer stats=Zone stats: {},Server stats: []}ServerList:null
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:38:51,529 ClientFactory.java (line 161) Client:middletier-client instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=middletier-client,current list of Servers=[pc54.home:9191],Load balancer stats=Zone stats: {defaultzone=[Zone:defaultzone;	Instance count:1;	Active connections count: 0;	Circuit breaker tripped count: 0;	Active connections per server: 0.0;]
},Server stats: [[Server:pc54.home:9191;	Zone:defaultZone;	Total Requests:0;	Successive connection failure:0;	Total blackout seconds:0;	Last connection made:Thu Jan 01 01:00:00 CET 1970;	First connection made: Thu Jan 01 01:00:00 CET 1970;	Active Connections:0;	total failure count in last (1000) msecs:0;	average resp time:0.0;	90 percentile resp time:0.0;	95 percentile resp time:0.0;	min resp time:0.0;	max resp time:0.0;	stddev resp time:0.0]
]}ServerList:DiscoveryEnabledNIWSServerList:; clientName:middletier-client; Effective vipAddresses:middletier; isSecure:false; datacenter:null
 INFO [hystrix-RSSThreadPool-1] 2017-04-09 21:38:51,533 ClientFactory.java (line 86) Client Registered:com.netflix.niws.client.http.RestClient@65f068d
 WARN [main] 2017-04-30 19:02:57,196 URLConfigurationSource.java (line 122) No URLs will be polled as dynamic configuration sources.
 INFO [main] 2017-04-30 19:02:57,460 URLConfigurationSource.java (line 123) To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
 INFO [main] 2017-04-30 19:02:57,485 DynamicPropertyFactory.java (line 271) DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@3cbbc1e0
 INFO [main] 2017-04-30 19:02:57,606 KaryonServer.java (line 146) Loading application properties with app id: edge and environment: null
 INFO [main] 2017-04-30 19:02:57,657 ConfigurationUtils.java (line 182) Loaded properties file jar:file:/home/polivier/Documents/AL/ESIR2_AL/recipes-rss/rss-edge/build/libs/rss-edge-0.1.0-SNAPSHOT.jar!/edge.properties
 INFO [main] 2017-04-30 19:02:59,799 Version.java (line 56) Hibernate Validator null
 INFO [main] 2017-04-30 19:02:59,829 ServerBootstrap.java (line 239) No health check invocation strategy specified, using the default strategy com.netflix.karyon.server.eureka.AsyncHealthCheckInvocationStrategy. In order to override this behavior you provide an implementation of com.netflix.karyon.server.eureka.HealthCheckInvocationStrategy and specify the fully qualified class name of the implementation in the property com.netflix.karyon.health.check.strategy
 INFO [main] 2017-04-30 19:02:59,866 AsyncHealthCheckInvocationStrategy.java (line 77) Application health check handler to be used by karyon: com.netflix.recipes.rss.server.EdgeHealthCheckHandler
 INFO [main] 2017-04-30 19:02:59,918 NativeMethodAccessorImpl.java (line -2) Logging to org.slf4j.impl.Log4jLoggerAdapter(org.mortbay.log) via org.mortbay.log.Slf4jLog
 WARN [main] 2017-04-30 19:03:00,226 PropertiesInstanceConfig.java (line 338) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 WARN [main] 2017-04-30 19:03:00,227 DefaultEurekaClientConfig.java (line 89) Cannot find the properties specified : eureka-client. This may be okay if there are other environment specific properties or the configuration is installed with a different mechanism.
 INFO [main] 2017-04-30 19:03:00,648 DiscoveryClient.java (line 553) Disable delta property : false
 INFO [main] 2017-04-30 19:03:00,648 DiscoveryClient.java (line 555) Application is null : false
 INFO [main] 2017-04-30 19:03:00,649 DiscoveryClient.java (line 557) Registered Applications size is zero : true
 INFO [main] 2017-04-30 19:03:00,649 DiscoveryClient.java (line 560) Application version is -1: true
 INFO [main] 2017-04-30 19:03:00,737 DiscoveryClient.java (line 616) Getting all instance registry info from the eureka server
 INFO [main] 2017-04-30 19:03:00,896 DiscoveryClient.java (line 624) The response status is 200
 INFO [main] 2017-04-30 19:03:00,898 DiscoveryClient.java (line 886) Starting heartbeat executor: renew interval is: 30
 INFO [main] 2017-04-30 19:03:00,909 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-30 19:03:00,981 Server.java (line 266) jetty-7.x.y-SNAPSHOT
 INFO [main] 2017-04-30 19:03:01,022 ContextHandler.java (line 744) started o.e.j.s.ServletContextHandler{/,null}
 INFO [main] 2017-04-30 19:03:01,050 AbstractConnector.java (line 338) Started SelectChannelConnector@0.0.0.0:9092
 INFO [main] 2017-04-30 19:03:01,051 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.AdminResourcesContainer
 INFO [main] 2017-04-30 19:03:01,052 ServerInitializer.java (line 74) Initializing component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-30 19:03:01,055 WebAdminComponent.java (line 46) Set the default page for admin container to: /adminres/webadmin/index.html
 INFO [main] 2017-04-30 19:03:01,056 ServerInitializer.java (line 77) Successfully initialized component class: com.netflix.adminresources.resources.WebAdminComponent
 INFO [main] 2017-04-30 19:03:01,056 ApplicationFinder.java (line 88) No application classes (Annotated with @Application) found. It is fine if you do not use the annotation model for applications.
 INFO [main] 2017-04-30 19:03:01,056 ServerInitializer.java (line 86) No application classes found, this is fine if the application class is not annotated.
 INFO [main] 2017-04-30 19:03:01,057 NativeMethodAccessorImpl.java (line -2) jetty-6.0.x
 INFO [main] 2017-04-30 19:03:01,260 NativeMethodAccessorImpl.java (line -2) Started SocketConnector @ 0.0.0.0:9090
 INFO [hystrix-RSSThreadPool-1] 2017-04-30 19:03:26,757 ConnectionPoolCleaner.java (line 110) Initializing ConnectionPoolCleaner for NFHttpClient:middletier-client
 INFO [hystrix-RSSThreadPool-1] 2017-04-30 19:03:26,863 BaseLoadBalancer.java (line 184) Client:middletier-client instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=middletier-client,current list of Servers=[],Load balancer stats=Zone stats: {},Server stats: []}ServerList:null
 INFO [hystrix-RSSThreadPool-1] 2017-04-30 19:03:26,892 ClientFactory.java (line 161) Client:middletier-client instantiated a LoadBalancer:DynamicServerListLoadBalancer:{NFLoadBalancer:name=middletier-client,current list of Servers=[pc54.home:9191],Load balancer stats=Zone stats: {defaultzone=[Zone:defaultzone;	Instance count:1;	Active connections count: 0;	Circuit breaker tripped count: 0;	Active connections per server: 0.0;]
},Server stats: [[Server:pc54.home:9191;	Zone:defaultZone;	Total Requests:0;	Successive connection failure:0;	Total blackout seconds:0;	Last connection made:Thu Jan 01 01:00:00 CET 1970;	First connection made: Thu Jan 01 01:00:00 CET 1970;	Active Connections:0;	total failure count in last (1000) msecs:0;	average resp time:0.0;	90 percentile resp time:0.0;	95 percentile resp time:0.0;	min resp time:0.0;	max resp time:0.0;	stddev resp time:0.0]
]}ServerList:DiscoveryEnabledNIWSServerList:; clientName:middletier-client; Effective vipAddresses:middletier; isSecure:false; datacenter:null
 INFO [hystrix-RSSThreadPool-1] 2017-04-30 19:03:26,897 ClientFactory.java (line 86) Client Registered:com.netflix.niws.client.http.RestClient@24084044
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-30 19:03:40,900 DiscoveryClient.java (line 1283) DiscoveryClient_UNKNOWN/pc54.home - retransmit instance info with status UP
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-30 19:03:40,900 DiscoveryClient.java (line 424) DiscoveryClient_UNKNOWN/pc54.home: registering service...
 INFO [DiscoveryClient_InstanceInfo-Replictor] 2017-04-30 19:03:40,900 DiscoveryClient.java (line 429) DiscoveryClient_UNKNOWN/pc54.home - registration status: not sent
